<?xml version="1.0" encoding="UTF-8"?>
<testsuite name="contrato_tenant-20240917152120" tests="1" file=".py" time="52.552" timestamp="2024-09-17T15:22:12" failures="1" errors="0" skipped="0">
	<testcase classname="contrato_tenant" name="test_contrato_tenant" time="52.552" timestamp="2024-09-17T15:22:12" file="Cuentacontrato.py" line="22">
		<failure type="AssertionError" message=""><![CDATA[Traceback (most recent call last):
  File "c:\Users\luist\repos\Automatizaciones_M-dulos-\Cuentacontrato.py", line 96, in test_contrato_tenant
    validate_text_visible(self.driver, element1, number_expected)
  File "c:\Users\luist\repos\Automatizaciones_M-dulos-\Elements.py", line 1186, in validate_text_visible
    assert text_obtained == text_expected
AssertionError
]]></failure>
		<system-out><![CDATA[¡Elemento encontrado y clickeado 10 veces con éxito!
¡Elemento encontrado y clickeado 1 veces con éxito!
El texto encontrado es  Estado De Mis Contratos
El texto es visible para el usuario
]]></system-out>
	</testcase>
</testsuite>
